source/stylesheets/case-studies/creators-school/_stripe.scss

```diff
@@ -29,6 +29,8 @@ $cs-Stripe-triangleHeight-mobile: 90px;
   &:after {
     content: '';
 
+    z-index: 500;
```

Comment:
shouldn't this be adding to one of the existing layer variables instead of a hardcoded value?